src/main/java/mServer/crawler/sender/phoenix/tasks/ZdfTaskBase.java

```diff
@@ -0,0 +1,70 @@
+package mServer.crawler.sender.phoenix.tasks;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import java.lang.reflect.Type;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import javax.ws.rs.client.Invocation.Builder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+import mServer.crawler.sender.MediathekReader;
+import mServer.crawler.sender.orf.CrawlerUrlDTO;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public abstract class ZdfTaskBase<T, D extends CrawlerUrlDTO> extends AbstractRestTask<T, D> {
+
+  private static final String HEADER_AUTHENTIFICATION = "Api-Auth";
+
+  private static final Logger LOG = LogManager.getLogger(ZdfTaskBase.class);
+
+  private final GsonBuilder gsonBuilder;
+
+  public ZdfTaskBase(MediathekReader aCrawler,
+          ConcurrentLinkedQueue<D> aUrlToCrawlDtos, Optional<String> aAuthKey) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDtos" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)